docs/4-querying-lists.md

```diff
@@ -17,6 +17,21 @@ const queryObservable = af.database.list('/items', {
 });
 ```
 
+**Query Options:**
+
+| method   | purpose            | 
+| ---------|--------------------| 
+| orderByChild | specify a child to order by |
```

Comment:
Each one of these methods should have backticks around them, such as `orderByChild`, `orderByKey`, etc...